packages/components/src/AsyncComponent/index.js

```diff
@@ -9,18 +9,22 @@ const BaseAsyncComponent = ({
     module,
     fallback,
     innerRef,
+    className,
     ...props
 }) => {
     return (
-        <ScalprumComponent
-            appName={appName}
-            module={module}
-            scope={scope || appName}
-            ErrorComponent={fallback}
-            ref={innerRef}
-            fallback={fallback}
-            {...props}
-        />
+        <section className={`${className || ''} ${appName}`}>
```

Comment:
What if we add a `component` prop (like `a`, `div`, `span`, etc.)  to allow the consumer to customize the element wrapper? And default to section. It would be useful if you are sharing just a component, not a whole app (like a card).

Also probably use `classnames` for the class. It's installed anyway.